Description:
 CLONE_NEWNS is only defined with _GNU_SOURCE (fixes FTBFS; lp: #935514)
Origin: upstream, http://fuse.git.sourceforge.net/git/gitweb.cgi?p=fuse/fuse;a=commitdiff;h=ae43094c13ecf49e0b738bbda633cf193c7b3670;hp=42d5c66b0bd1b465ddd586a8c339b56799694f3f

Index: fuse-2.8.6/util/fusermount.c
===================================================================
--- fuse-2.8.6.orig/util/fusermount.c	2012-02-27 14:57:29.000000000 +0100
+++ fuse-2.8.6/util/fusermount.c	2012-02-27 15:03:20.000000000 +0100
@@ -7,6 +7,7 @@
 */
 /* This program does the mounting and unmounting of FUSE filesystems */
 
+#define _GNU_SOURCE /* for clone */
 #include <config.h>
 
 #include "mount_util.h"
